validator/client/src/main/java/tech/pegasys/teku/validator/client/KeyManager.java

```diff
@@ -13,12 +13,73 @@
 
 package tech.pegasys.teku.validator.client;
 
+import java.util.List;
 import java.util.Set;
+import org.apache.commons.lang3.NotImplementedException;
 import tech.pegasys.teku.bls.BLSPublicKey;
 import tech.pegasys.teku.validator.client.loader.ValidatorLoader;
+import tech.pegasys.teku.validator.client.restapi.apis.schema.ActiveValidator;
+import tech.pegasys.teku.validator.client.restapi.apis.schema.DeleteKeysResponse;
+import tech.pegasys.teku.validator.client.restapi.apis.schema.ImportStatus;
 
 public class KeyManager {
 
+  /**
+   * Get a listing of active validator keys
+   *
+   * @return a list of active validators
+   */
+  List<ActiveValidator> getActiveValidatorKeys() {
```

Comment:
Should we set this method as public?

Reply:
copied from interface, so yep probably shoudl be